src/mlpack/bindings/cli/cli_option.hpp

```diff
@@ -131,59 +131,59 @@ class CLIOption
       #undef BASH_RED
       #undef BASH_CLEAR
 
-      outstr << "Parameter --" << boostName << " (" << data.alias << ") "
+      outstr << "Parameter --" << cliName << " (" << data.alias << ") "
              << "is defined multiple times with the same identifiers."
              << std::endl;
     }
 
-    CLI::Add(std::move(data));
+    IO::Add(std::move(data));
 
     // Set some function pointers that we need.
-    CLI::GetSingleton().functionMap[tname]["DefaultParam"] =
+    IO::GetSingleton().functionMap[tname]["DefaultParam"] =
         &DefaultParam<N>;
-    CLI::GetSingleton().functionMap[tname]["OutputParam"] =
+    IO::GetSingleton().functionMap[tname]["OutputParam"] =
         &OutputParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetPrintableParam"] =
+    IO::GetSingleton().functionMap[tname]["GetPrintableParam"] =
         &GetPrintableParam<N>;
-    CLI::GetSingleton().functionMap[tname]["StringTypeParam"] =
+    IO::GetSingleton().functionMap[tname]["StringTypeParam"] =
         &StringTypeParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetParam"] = &GetParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetRawParam"] = &GetRawParam<N>;
-    CLI::GetSingleton().functionMap[tname]["AddToPO"] = &AddToPO<N>;
-    CLI::GetSingleton().functionMap[tname]["MapParameterName"] =
+    IO::GetSingleton().functionMap[tname]["GetParam"] = &GetParam<N>;
+    IO::GetSingleton().functionMap[tname]["GetRawParam"] = &GetRawParam<N>;
+    IO::GetSingleton().functionMap[tname]["AddToPO"] = &AddToPO<N>;
+    IO::GetSingleton().functionMap[tname]["MapParameterName"] =
         &MapParameterName<N>;
-    CLI::GetSingleton().functionMap[tname]["SetParam"] = &SetParam<N>;
-    CLI::GetSingleton().functionMap[tname]["GetPrintableParamName"] =
+    IO::GetSingleton().functionMap[tname]["SetParam"] = &SetParam<N>;
```

Comment:
For some reason I can't comment on the existing thread.

> It is still used when generating python code in print_input_processing.hpp
I am not sure why it is used in Python and Julia, but it is still used there.

Yeah, I just meant from the CLI bindings; I think it is safe to remove them only from there. :+1: